### DISCUSSION TITLE
Converting to private repo

### DISCUSSION THREAD

**Discussion by MarkBauer** [community/community#31785] [Other Feature Feedback, Questions, & Ideas]
Hi there! Thank you in advance for bearing with me and my super newbie question...

I have a free account and I use Github pages for my portfolio site. I want the site to remain accessible to the public, but I would like to make the repo private. Basically I don't want people to still be able to access the site but I don't want the source code viewable on my Github profile. 

When I go to settings, the control to make the repo private lives in the "Danger Zone" section. There's a message warning me that this is a "potentially destructive action". Can someone tell me what will happen to my site if I change the repo from public to private? I'm gun shy about clicking that button because I don't want to screw anything up on my site.

**Comment by radar-blip** [comment]
It will just make it private, and nobody will be able to see it. Except you of course.
Your site will be fine, just out of view.

**Reply by MarkBauer** [reply]
Thank you very much, @linuxlawson!

**Reply by radar-blip** [reply]
You're welcome! 👍🏻

**Comment by DenverCoder1** [comment]
GitHub Pages sites will be public even if the repo is private.

So if you were to make the repo private, people will still be able to access the site.

**Edit:** This only applies to repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server.

See https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages

**Reply by jfullstackdev** [reply]
if it's not a free account 

![image](https://user-images.githubusercontent.com/47092464/188263040-0f5cd9ba-3741-4dc0-9060-cd32b8d0311a.png)

**Comment by jfullstackdev** [comment]
> I have a free account and I use Github pages for my portfolio site. I want the site to remain accessible to the public, but I would like to make the repo private. Basically I don't want people to still be able to access the site but I don't want the source code viewable on my Github profile.

it can't be for a free account, you need to pay for that 

![image](https://user-images.githubusercontent.com/47092464/188263047-f55229ca-b3b6-4483-90fb-16b1c3900eb7.png)
